scripts/build.sh

```diff
@@ -192,6 +203,17 @@ main() {
     rm -rf "${build_dir}"
   fi
 
+  if [[ -n "$plugins" ]] && [[ -z `command -v lpm` ]]; then
+    mkdir -p "${build_dir}"
+    lpm_path="$(pwd)/${build_dir}/lpm$(get_executable_extension)"
+    if [[ ! -e "$lpm_path" ]]; then
+      curl --insecure -L -o "$lpm_path" \
+        "https://github.com/lite-xl/lite-xl-plugin-manager/releases/download/${LPM_VERSION:-v1.3.1}/lpm.$(get_platform_tuple)$(get_executable_extension)"
```

Comment:
I wonder if for the default case we should use `latest`.

Reply:
I think this version is the same as what build box has. Also helps with any future regressions and stuff.

Reply:
It was mostly to avoid having another place to update every so often.
But if you think it's alright I'm ok with it.

Reply:
Sure. This code path wouldn't get triggered on CI build anyway. Might as well let the user have the latest stuff.